### DISCUSSION TITLE
QR code login

### DISCUSSION THREAD

**Discussion by jakelala** [Johnserf-Seed/TikTokDownload#552] [Q&A]
This QR code looks like it cannot be scanned to log in. What is the problem?
![IMG20230919195713](https://github.com/Johnserf-Seed/TikTokDownload/assets/60954110/1321cd95-848e-442b-8e7a-d048611dbcc3)

**Comment by chengazhen** [comment]
我这边实现了一个客户端应用，但是扫码登录还在开发。发布的版本是吧 web端实现为了客户端； https://github.com/Journey98/TikTokWeb-electron

**Reply by outofsky** [reply]
安装最新版客户端即可。https://github.com/microsoft/terminal

**Comment by Johnserf-Seed** [comment]
The first thing to do is to look at the readme in detail
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/40727745/4628055b-df64-4dc0-904c-bafc9a138385)
